x-pack/plugins/security_solution/public/flyout/document_details/shared/hooks/use_highlighted_fields.test.tsx

```diff
@@ -126,13 +126,13 @@ describe('useHighlightedFields', () => {
     const hookResult = renderHook(() =>
       useHighlightedFields({
         dataFormattedForFieldBrowser: dataFormattedForFieldBrowser.concat({
-          category: 'crowdstrike',
-          field: 'crowdstrike.event.DeviceId',
+          category: 'device',
```

Comment:
same here and below. 

(also: maybe you remove these test level mocks and replace it with the one we now have for testing under `endpointAlertDataMocks`?